src/Components/Todos.jsx

```diff
@@ -0,0 +1,45 @@
+import React, { Fragment, useState } from "react";
+import NewTodo from "Components/NewTodo";
+import Todo from "Components/Todo";
+import { useSelector } from "react-redux";
+
+export default function Todos() {
+  const todos = useSelector((state) => state.todos);
+
+  const [showNewTodoForm, setShowNewTodoForm] = useState(false);
```

Comment:
Are we showing form? or just a card? Can you please look up for the name?

Reply:
showing the form